realm/realm-library/src/androidTest/java/io/realm/entities/AllTypes.java

```diff
@@ -47,6 +50,8 @@
     private Date columnDate = new Date(0);
     @Required
     private byte[] columnBinary = new byte[0];
+    @Ignore
```

Comment:
That seems excessive, since none of the tests will pass, as long as that is in place...